readme.md

```diff
@@ -1,11 +1,17 @@
-Babylon.js - Create Your Own Shader (CYOS)
-==========================================
+Evolution of Babylon.js - Create Your Own Shader (CYOS) for all 3D Web APIs
+============================================================================
 
-Live version: http://www.babylonjs.com/cyos/
+Original live version: http://www.babylonjs.com/cyos/
+
+I have modified CYOS code to allow for dynamic selection of WebGL 1.0, 2.0 and WebGPU.
+
+I have updated the code for modern JavaScript with modules, updated 3rd party dependencies, removed dead code.
```

Comment:
I love your contributions! And you should have all the credit for that :-) But if you can first describe the basic functionality and then later say something like "modified by ..." it would be great. The readme should be as abstract as possible

Reply:
Likewise, I should have used another branch for the PR. Reverting the Readme to the original one.